Core/Object Arts/Dolphin/IDE/Base/Development System.pax

```diff
@@ -1982,13 +1983,21 @@ browse
     
     	^Smalltalk developmentSystem browseMethod: self!
 
+isDeprecated
```

Comment:
I see you've moved this method to the development system package (which is where it should have been in the first place), but it may as well become SmalltalkSystem>>isDeprecatedMethod:. Even though it was public before I don't think anyone will have called it.